great_expectations/expectations/core/expect_column_pair_values_to_be_equal.py

```diff
@@ -56,6 +65,88 @@ class ExpectColumnPairValuesToBeEqual(ColumnPairMapExpectation):
         An [ExpectationSuiteValidationResult](https://docs.greatexpectations.io/docs/terms/validation_result)
 
         Exact fields vary depending on the values passed to result_format, catch_exceptions, and meta.
+
+    Supported Datasources:
```

Comment:
```suggestion
    Supported Data Sources
```
Again, locate all instances and make them consistent.